.circleci/docker-compose.yml

```diff
@@ -0,0 +1,155 @@
+version: '3.3'
```

Comment:
We've copied `docker-compose.yml` (with a few changes) from the `uwcirg/tb-api` repo.

On the surface, this just lets us keep our integration tests local with the frontend code.

Digging a little deeper, this setup gives us a bit more stability;
if we introduce additional client apps, each one can define their own `docker-compose` configurations according to their needs.
The server codebases shouldn't have opinions on how the client applications are set up.